README.md

```diff
@@ -1,13 +1,16 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project was part of Technigo Bootcamp, and the assignment this week was to build an API with Express from a set datafile. This was our first project working in the backend, and the requirements included creating a few routes both for a collection of results and a specific result.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I am a big Netflix fan so I decided to use the data file for Netflix data. I found the assignment suprisingly easy to do and manage to do all the basic requirements in day 1 and was then able to create a front end to try and use my API to display it.
+
+I created 4 endpoints, including one with all TV-shows, one for a specific show and then I also did one that gives a random show from the data file. I also added some queries so that one can filter on country and year.
+
+The endpoint to generate a random response triggered an idea to build a frontend which allows the user who want some ideas of what to watch to press a button to recive a random show.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+Find my API here: https://siris-tvshows-api.herokuapp.com/
+And my random tv-show generator here: https://sleepy-bassi-2f0ba1.netlify.app/
```

Comment:
It would be very nice to have some instructions, how to use your API. Remember to make documentation on what endpoints and queries your user can use. 🤓